app/views/image/show_image.html.erb

```diff
@@ -216,18 +216,22 @@
                   str1 = image_vote_as_short_string(value)
                   str2 = image_vote_as_help_string(value)
                   str3 = :image_show_vote_and_next.t(value: str1)
-                  css_class = "btn btn-default btn-block"
-                  css_class += " active" if current == value %>
+                  css_class = ""
+                  css_class = "bold" if current == value %>
```

Comment:
Just getting rid of those big huge ugly vote buttons and replacing them with links.